core/src/main/resources/i18n/displayStrings.properties

```diff
@@ -1097,13 +1097,23 @@ support.filter=Search disputes
 support.filter.prompt=Enter trade ID, date, onion address or account data
 support.prompt.signedOffer.penalty.msg=This will charge the buyer as maker the trade fee of {0} XMR and return the remaining trade funds of {1} XMR to their wallet. Are you sure you want to send?\n\n\
   Other Info:\n\
-  Transaction Fee: {2}\n\
+  Transaction Fee: {2} XMR\n\
   Reserve Tx Hash: {3}\n\
   Reserve Tx Key Images: {4}\n\
 
 support.contextmenu.penalize.msg=Penalize {0} by publishing reserve tx
 support.prompt.signedOffer.error.msg=Trade record does not exist; contact administrator.
-support.info.submitTxHex=Reserve transaction has been published with the following result: {0}
+support.info.submitTxHex=Reserve transaction has been published with the following result:\n
+support.result.hasInvalidInput=Transaction has invalid inputs\n
```

Comment:
Let's not break down the submission result for formatting - too much potential maintenance for an error message which might change, and unnecessary structure when the error can be printed.

After clicking the confirmation screen, they should see either a success message (if submitting did not throw and isGood() is true), or an error message with the exception message or JSON serialized submission message.